[OSDOCS#17059]: CQA 2.0 updating disconnected cluster OSUS (#113825)
openshift/openshift-docs: modules/config-access-for-sec-reg-osus.adoc

// Module included in the following assemblies:
//
// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

:_mod-docs-content-type: PROCEDURE
[id="registry-configuration-for-update-service_{context}"]
= Configuring access to a secured registry for the OpenShift Update Service

[role="_abstract"]
If your release images are contained in a registry whose HTTPS X.509 certificate is signed by a custom certificate authority, you must configure additional trust stores for image registry access for the update service.

.Prerequisites
* You must have the `oc` command-line interface (CLI) tool installed.
* You must provision a container image registry in your environment with the container images for your update.

.Procedure

> . Complete the steps in 'Configuring additional trust stores for image registry access' along with the following changes for the update service.

Reviewer: This is good, I think we just tend to use double quotes for references to content sections: 
```suggestion
. Complete the steps in "Configuring additional trust stores for image registry access" along with the following changes for the update service.
```

+
.Image registry CA config map example for the update service
[source,yaml]
----
apiVersion: v1
kind: ConfigMap
metadata:
  name: my-registry-ca
data:
  updateservice-registry: |
    -----BEGIN CERTIFICATE-----
    ...
    -----END CERTIFICATE-----
  registry-with-port.example.com..5000: |
    -----BEGIN CERTIFICATE-----
    ...
    -----END CERTIFICATE-----
----
+
where: 

> `data.updateservice-registry`:: Specifies the required config map key name containing the registry certificate authority (CA) certificate. Do not alter this key string.

Reviewer: Unless I missed something, I think this last sentence is new, and I wouldn't add it unless it was for a specific reason. 
```suggestion
`data.updateservice-registry`:: Specifies the required config map key name containing the registry certificate authority (CA) certificate.
```

> `data.registry-with-port.example.com..5000`:: Specifies a registry string that includes a port number. Replace the colon (`:`) with two periods (`..`) to comply with config map key validation rules.

Reviewer: This one is tough, it kind of forces you to rewrite a bit of original content to make sense with the description list format, and you no longer mention the example registry URL. I would consider actually making this a NOTE:

```suggestion
+
[NOTE]
====
If a registry has the port, such as `registry-with-port.example.com:5000`, `:` should be replaced with `..`.
====
```

This way the original text is better preserved. But your way is reasonable as well. What do you think?

Author: I agree. I doubted about adding it as a callout anyway, but did not want to make something that doesn't make sense. This helps. Thanks.
